README.md

```diff
@@ -110,7 +110,18 @@ Users and teams managing tenant apps should not have permissions to change clust
 ## Deploy the blueprint
 
 1. Open [Cloud Shell](https://cloud.google.com/shell)
-1. Clone this repository
+
+1. Initialize the local repository where the environment configuration will be
+   stored:
+
+   ```sh
+   ACM_REPOSITORY_PATH= # Path on the host running Terraform to store environment configuration
```

Comment:
As we get rind of reference to Anthos, maybe rename these variables to CONFIG_SYNC ?
https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs

Reply:
Makes sense, although we can track it in a separate PR because there are many references `acm_...` variables.